### PR TITLE
Add memory.workingset.limit.pct field in container and pod data stream

### LINKED ISSUE #2374: Add memory.workingset.limit.pct and memory.working_set.limit.pct` fields in pod and container kubernetes integration
Following [#29547](https://github.com/elastic/beats/pull/29547) and the addition of `memory.working_set.limit.pct` for kubernetes pod metricset and `memory.workingset.limit.pct` for kubernetes container metricset, these fields have to be added in the `pod` and `container`
Metricset of `kubernetes integration`.

### PR DESCRIPTION
**Pull request by MichaelKatsoulis**
## What does this PR do?

This PR adds `memory.working_set.limit.pct` field  in kubernetes pod data stream and `memory.workingset.limit.pct` in kubernetes container data stream following the addition of those fields in the k8s module metrcisets by [#29547 ](https://github.com/elastic/beats/pull/29547)

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/master/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [ ] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## How to test this PR locally

Bring up elastic stack version 8.1.0-SNAPSHOT and enable kubernetes integration. 
The new fields with the correct types are added in the `metrics-kubernetes.pod` and `metrics-kubernetes.container` index templates.

## Related issues

- Closes #2374

## Screenshots

pod mapping
container_mapping

### CODE REVIEW

File: packages/kubernetes/data_stream/container/fields/fields.yml [hunk 1]
```diff
@@ -146,6 +146,11 @@
               metric_type: gauge
               description: |
                 Working set memory usage
+            - name: limit.pct
+              type: scaled_float
+              format: percent
```
Comment (tetianakravchenko) [inline packages/kubernetes/data_stream/container/fields/fields.yml:151]: should here be added `unit` as well as `metric_type`? according to https://github.com/elastic/observability-dev/blob/main/docs/dc/integrations/development/generic_guidelines.md#document-all-fields

File: packages/kubernetes/data_stream/pod/fields/fields.yml [hunk 2]
```diff
@@ -111,6 +111,11 @@
               metric_type: gauge
               description: |
                 Total working set memory
+            - name: limit.pct
+              type: scaled_float
+              format: percent
```
Comment (tetianakravchenko) [inline packages/kubernetes/data_stream/pod/fields/fields.yml:116]: the same here

File: packages/kubernetes/data_stream/container/fields/fields.yml (hunk 1 above)
Comment (MichaelKatsoulis) [reply packages/kubernetes/data_stream/container/fields/fields.yml:151]: You are right. Added in f0e4319babfcb3a166e6b1bd85d8161d398b6226

**Review by tetianakravchenko — APPROVED** [review]

File: packages/kubernetes/data_stream/container/fields/fields.yml (hunk 1 above)
Comment (tetianakravchenko) [reply packages/kubernetes/data_stream/container/fields/fields.yml:151]: @MichaelKatsoulis should also `sample_event.json` files also get updated with new fields?

Comment (MichaelKatsoulis) [reply packages/kubernetes/data_stream/container/fields/fields.yml:151]: Yes It should! 2c918e38c484bc7486f3948ede41d1ec8746b431

**Pull request merged**